.github/workflows/semver-checks.yml

```diff
@@ -37,17 +37,19 @@ jobs:
         run: cargo binstall cargo-semver-checks@0.32.0 --no-confirm
       - name: "Run semver checker script"
         run: ./contrib/check-semver-pr.sh
-      - name: "Add PR label to breaking changes"
-        uses: actions-ecosystem/action-add-labels@v1
+      - name: "Save PR Number"
         if: ${{ hashFiles('semver-break') != '' }}
-        with:
-          labels: "API break"
-      - name: Comment PR
-        uses: thollander/actions-comment-pull-request@v2
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          mkdir -p ./pr-semver-break
+          echo $PR_NUMBER > ./pr-semver-break/semver-break-NR
+      - name: "Upload PR Number"
```

Comment:
This looks like the example from the article copied verbatim. It should be adapted to our needs, specifically, we should upload the status, not the PR number. We need to get the PR number from the even payload, otherwise griefers could make this post a comment on *all PRs* which would be highly annoying.